### PR TITLE
Add comments to code snippets

### LINKED ISSUE #336: HTML snippets should have comments
Spotted in user testing.

While copy & pasting code snippets, for example, the layout ones, it's not clear the purpose of every tag. We found the user copying the whole block instead of reading which element he should use.

We need to provide that context so what we've thought is to add comments to those code snippets to highlight what is optional and it's there because of the snippet.

### PR DESCRIPTION
**Pull request by IagoLast**
Fix #336 
---

### CODE REVIEW

File: packages/styles/src/core/layout/main/map-area/README.md [hunk 1]
```diff
@@ -12,10 +12,11 @@ The most basic usage of the `as-map-wrapper` element is when you only need to sh
 ```html
 <div class="as-app" style="width:100%; height:300px;">
     <main class="as-content">
+      <!-- element code starts here -->
```
[the remark this answers is not in the document]
Comment (IagoLast) [reply packages/styles/src/core/layout/main/map-area/README.md]: cc @ivanmalagon

File: packages/styles/src/core/layout/main/map-area/README.md [hunk 2]
```diff
@@ -14,10 +14,16 @@ showSource: true
 ---
 <div class="as-app" style="width:100%; height:300px;">
     <main class="as-content">
+      <!-- element code starts here -->
       <div class="as-map-area as-bg--badge-blue">
+<<<<<<< HEAD
```
Comment (ivanmalagon) [inline packages/styles/src/core/layout/main/map-area/README.md]: <<<< HEAD 👱

File: packages/styles/src/core/layout/main/map-area/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ivanmalagon) [reply packages/styles/src/core/layout/main/map-area/README.md]: I'd put the element name.

`

File: packages/styles/src/core/layout/main/map-area/map-panels/README.md [hunk 3]
```diff
@@ -34,9 +34,9 @@ showSource: true
   <div class="as-content">
     <main class="as-main as-bg--badge-blue">
       <div class="as-map-area">
-      <!-- CARTO.js or CARTO-VL will place the map on this div  -->
-      <div id="map"></div>
+      <div id="map"></div> 
       <div class="as-map-panels">
+        <!-- element code starts here -->
```
Comment (ivanmalagon) [inline packages/styles/src/core/layout/main/map-area/map-panels/README.md]: Same formula as above.

`<!-- as-panel example code starts here -->`

File: packages/styles/src/core/layout/main/map-area/map-panels/README.md [hunk 4]
```diff
@@ -64,14 +65,14 @@ showSource: true
   <div class="as-content">
     <main class="as-main as-bg--badge-blue">
       <div class="as-map-area">
-        <!-- CARTO.js or CARTO-VL will place the map on this div  -->
         <div id="map"></div>
         <div class="as-map-panels">
-
           <div class="as-panel as-panel--middle as-panel--center">
+            <!-- element code starts here -->
```
Comment (ivanmalagon) [inline packages/styles/src/core/layout/main/map-area/map-panels/README.md]: `<!-- as-panel___element ...`

File: packages/styles/src/core/layout/main/map-area/map-panels/README.md [hunk 5]
```diff
@@ -104,7 +103,7 @@ showSource: true
               <p class="as-body"> Panel 1 </p>
             </div>
           </div>
-
+          <!-- element code starts here -->
```
Comment (ivanmalagon) [inline packages/styles/src/core/layout/main/map-area/map-panels/README.md]: ... and so on.

**Pull request merged**